docs/test/package.json

```diff
@@ -4,12 +4,10 @@
   "description": "",
   "author": "",
   "license": "ISC",
-  "devDependencies": {
-    "vectordb": "https://gitpkg.now.sh/lancedb/lancedb/node?main"
-  },
   "dependencies": {
     "fs": "^0.0.1-security",
     "glob": "^10.2.7",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "vectordb": "https://gitpkg.now.sh/lancedb/lancedb/node?main"
```

Comment:
Curious why vectordb was dev dependency before

Reply:
Mistake on my part, should have changed it to dev, was doing some experimenting